### PR TITLE
Use apikey in body for mailchimp

### PR DESCRIPTION
**Pull request by freaz**
## Description

I just learnt we already support apikey in body. So integration parameter isn't needed.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: capabilities/communication/send-email/maps/mailchimp.suma [hunk 1]
```diff
@@ -6,7 +6,7 @@ provider = "mailchimp"
 
 map SendEmail {
   body = {
-    key: input.mailchimp_api_key,
+    key: parameters.API_KEY,
```
Comment (janhalama) [inline capabilities/communication/send-email/maps/mailchimp.suma:9]: Key parameter is added to the body through security scheme and `API_KEY` integration parameter is not defined in mailchimp provider. This line should be omitted.

**Review by janhalama — COMMENTED** [review]
Looks good, found just one issue.

File: capabilities/communication/send-email/maps/mailchimp.suma (hunk 1 above)
Comment (freaz) [reply capabilities/communication/send-email/maps/mailchimp.suma:9]: Thats leftover. When I learned security schemes works with bodies as well. Will remove it. Thanks.

Comment (freaz) [reply capabilities/communication/send-email/maps/mailchimp.suma]: Fixed

**Review by janhalama — APPROVED** [review]
👍

**Pull request merged**